erizo_controller/erizoClient/src/utils/L.Resizer.js

```diff
@@ -1,460 +1,462 @@
-/* globals $$, Elements */
-'use strict';
+/* globals $$, jQuery, Elements, document, window */
+
+
+/**
+* Copyright 2013 Marc J. Schmidt. See the LICENSE file at the top-level
+* directory of this distribution and at
+* https://github.com/marcj/css-element-queries/blob/master/LICENSE.
+*/
+this.L = this.L || {};
+const L = this.L;
+
 /**
- * Copyright 2013 Marc J. Schmidt. See the LICENSE file at the top-level
- * directory of this distribution and at
- * https://github.com/marcj/css-element-queries/blob/master/LICENSE.
+ * @param {HTMLElement} element
+ * @param {String}      prop
+ * @returns {String|Number}
  */
-(function() {
-
-    this.L = this.L || {};
-    var L = this.L;
-
-    /**
-     *
-     * @type {Function}
-     * @constructor
-     */
-    L.ElementQueries = function() {
-        /**
-         *
-         * @param element
-         * @returns {Number}
-         */
-        function getEmSize(element) {
-            if (!element) {
-                element = document.documentElement;
-            }
-            var fontSize = getComputedStyle(element, 'fontSize');
-            return parseFloat(fontSize) || 16;
-        }
+function getComputedStyle(computedElement, prop) {
+  if (computedElement.currentStyle) {
+    return computedElement.currentStyle[prop];
+  } else if (window.getComputedStyle) {
+    return window.getComputedStyle(computedElement, null).getPropertyValue(prop);
```

Comment:
I get an infinite loop here. Disabling it and making the other adjustments (constructor) works.